insights/specs/__init__.py

```diff
@@ -207,71 +207,71 @@ class Specs(SpecSet):
     lsinitrd_lvm_conf = RegistryPoint()
     lsmod = RegistryPoint()
-    lsof = RegistryPoint()
+    lsof = RegistryPoint(filterable=True)
     lspci = RegistryPoint()
     lspci_kernel = RegistryPoint()
     lssap = RegistryPoint()
     lsscsi = RegistryPoint()
     ls_sys_firmware = RegistryPoint()
     ls_var_lib_mongodb = RegistryPoint()
-    ls_usr_sbin = RegistryPoint()
+    ls_usr_sbin = RegistryPoint(filterable=True)
     ls_var_log = RegistryPoint()
     ls_var_www = RegistryPoint()
     ls_var_spool_clientmq = RegistryPoint()
-    ls_var_tmp = RegistryPoint()
+    ls_var_tmp = RegistryPoint(filterable=True)
     ls_var_run = RegistryPoint()
     ls_var_spool_postfix_maildrop = RegistryPoint()
     ls_osroot = RegistryPoint()
     lvdisplay = RegistryPoint()
-    lvm_conf = RegistryPoint()
+    lvm_conf = RegistryPoint(filterable=True)
     lvs_noheadings = RegistryPoint()
     lvs_noheadings_all = RegistryPoint()
     lvs = RegistryPoint()
     machine_id = RegistryPoint()
     manila_conf = RegistryPoint()
-    mariadb_log = RegistryPoint()
+    mariadb_log = RegistryPoint(filterable=True)
     md5chk_files = RegistryPoint()
     mdstat = RegistryPoint()
     meminfo = RegistryPoint()
-    messages = RegistryPoint()
+    messages = RegistryPoint(filterable=True)
     metadata_json = RegistryPoint(raw=True)
     mlx4_port = RegistryPoint()
     modprobe = RegistryPoint(multi_output=True)
     module = RegistryPoint()
-    mongod_conf = RegistryPoint(multi_output=True)
+    mongod_conf = RegistryPoint(multi_output=True, filterable=True)
     mount = RegistryPoint()
     multicast_querier = RegistryPoint()
     multipath_conf = RegistryPoint()
     multipath__v4__ll = RegistryPoint()
     mysqladmin_vars = RegistryPoint()
     mysql_log = RegistryPoint(multi_output=True)
     mysqld_limits = RegistryPoint()
-    named_checkconf_p = RegistryPoint()
+    named_checkconf_p = RegistryPoint(filterable=True)
     netconsole = RegistryPoint()
     netstat_agn = RegistryPoint()
     netstat_i = RegistryPoint()
     netstat = RegistryPoint()
     netstat_s = RegistryPoint()
     networkmanager_dispatcher_d = RegistryPoint(multi_output=True)
     neutron_conf = RegistryPoint()
-    neutron_l3_agent_log = RegistryPoint()
-    neutron_metadata_agent_ini = RegistryPoint()
-    neutron_metadata_agent_log = RegistryPoint()
-    neutron_ovs_agent_log = RegistryPoint()
+    neutron_l3_agent_log = RegistryPoint(filterable=True)
+    neutron_metadata_agent_ini = RegistryPoint(filterable=True)
+    neutron_metadata_agent_log = RegistryPoint(filterable=True)
+    neutron_ovs_agent_log = RegistryPoint(filterable=True)
     neutron_plugin_ini = RegistryPoint()
-    neutron_server_log = RegistryPoint()
+    neutron_server_log = RegistryPoint(filterable=True)
     nfnetlink_queue = RegistryPoint()
-    nfs_exports_d = RegistryPoint(multi_output=True)
-    nfs_exports = RegistryPoint()
+    nfs_exports_d = RegistryPoint(multi_output=True, filterable=True)
```

Comment:
Any reason we add filters to the `nfs_exports_d` and `nfs_exports` specs?

Reply:
@lonicerae there are filters added for the 2 specs in some security plugins, then we have to add filters in insights-rules to make sure the expected lines will be collected.

Reply:
@csams @lonicerae  - After checking the related rules in security rules, I believe the filters should be removed. And the two specs (`nfs_exports` and `nfs_exports_d`) should be filtered.

https://gitlab.cee.redhat.com/insights-open-source/insights-prodsec-rules/blob/master/prodsec/rules/hardening_nfs.py
https://gitlab.cee.redhat.com/insights-open-source/insights-prodsec-rules/blob/master/prodsec/rules/hardening_nfs_world_access.py

Reply:
Removed